sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import scipy.sparse as sp
 import joblib
+import pytest
```

Comment:
The only file we can not use `pytest` in.

Reply:
uugghhh

should I just create a new test_utils_deprecations.py file?